lib/src/components/material_popup/material_popup.html

```diff
@@ -16,8 +16,8 @@
        [style.transform-origin]="transformOrigin"
        [class.visible]="showPopup">
       <div class="popup"
-           [style.max-height.px]="contentHeight"
-           [style.max-width.px]="contentWidth">
+           [style.max-height.px]="maxHeight"
```

Comment:
Should the removal of contentHeight/Width be mentioned in the changelog? Break?

Reply:
Done.